infra/scripts/lakekeeper.py

```
@@ -0,0 +1,143 @@
+        openid_provider_url: str,
+        client_id: str,
+        client_secret: str,
+        scope: str,
+    ) -> None:
+        """
+        Args:
+
+          server_url: URL for Lakekeeper instance
+        """
+        self._server_url = server_url
+        openid_config_resp = requests.get(
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)